### DISCUSSION TITLE
Is Windows 11 a supported OS for self-hosted runners?

### DISCUSSION THREAD

**Discussion by Rod-at-DOH** [community/community#80940] [Actions] [answered]
### Select Topic Area

Question

### Body

According to the document page [About self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) Windows 10 64-bit is one of the Windows OSs that support GitHub self-hosted runners. I presume that Windows 11 64-bit would work as well. 

Is that correct?

**Comment by ananuness** [comment, marked as answer]
As you saw, they only support 7, 8, 10 and windows servers, you can just check out which windows server do you have. Type `systeminfo` in your powershell and check the *OS Name*, if it doesn't work, you can check this [video](https://www.youtube.com/watch?v=CCISaVFnK_s) to see others methods!

**Comment by ghost** [comment]
how i can approvals my email address in the websites

**Comment by Xelliese** [comment]
Updates have included it as a supported operating system now. I know a lot of people were not updating to W11 because of this. I still don't like it and I bought a windows 10 license. [Click here](https://keyzon.com/products/microsoft-windows-10_pqslz/) if you need an affordable windows license.

**Reply by Rod-at-DOH** [reply]
I'm glad to hear this!
